ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/name/vch-creation-wizard-name.component.ts

```diff
@@ -38,7 +38,7 @@ export class VchCreationWizardNameComponent implements OnInit {
     // and not containing any invalid characters
     this.form = formBuilder.group({
       name: [
-        '', // TODO: implement a function to load existing data
+        'virtual-container-host', // TODO: implement a function to load existing data
```

Comment:
Is this TODO still needed? I'm not quite clear what existing data it is referring to.

Reply:
Neither do I. @jooskim? Is OK to remove this?

Reply:
that could now be removed. i put a todo there a long time ago after having a conversation with ben corrie that it'd be nice for the wizard to continue from where the user left. we don't have a concrete story or mockup yet so it's ok to remove it